openfisca_france/decompositions/fiche_de_paie_decomposition.xml

```diff
@@ -31,8 +31,7 @@
             <NODE code="contribution_developpement_apprentissage" color="249,208,211" desc="Contribution au Développement de l'apprentissage" shortname="Dév. apprenti." />
             <NODE code="contribution_solidarite_autonomie" color="249,208,211" desc="Contribution solidarite autonomie" shortname="CSA" />
             <NODE code="contribution_supplementaire_apprentissage" color="249,208,211" desc="Contribution supplémentaire à l'apprentissage" shortname="Suppl. apprenti." />
-            <NODE code="fnal_tranche_a" color="249,208,211" desc="FNAL" shortname="FNAL" />
-            <NODE code="fnal_tranche_a_plus_20" color="249,208,211" desc="FNAL +20" shortname="FNAL +20" />
+            <NODE code="fnal" color="249,208,211" desc="FNAL" shortname="FNAL" />
```

Comment:
`desc` should be "Fonds National Action Logement". I canot add to this PR, so I will do it after the merge if this is not fixed before.